docs/ref/request-response.txt

```diff
@@ -425,20 +425,27 @@ Methods
     Returns ``True`` if the request is secure; that is, if it was made with
     HTTPS.
 
-.. method:: HttpRequest.accepts(mime_type)
+.. method:: HttpRequest.get_preferred_type(media_types)
 
-    Returns ``True`` if the request ``Accept`` header matches the ``mime_type``
-    argument:
+    .. versionadded:: 5.2
+
+    Returns the preferred mime type from ``media_types``, based on the
+    ``Accept`` header, or ``None`` if the client does not accept any of the
+    provided types.
+
+    Assuming the client sends an ``Accept`` header of
+    ``text/html,application/json;q=0.8``:
 
     .. code-block:: pycon
 
-        >>> request.accepts("text/html")
-        True
+        >>> request.get_preferred_type(["text/html", "application/json"])
+        "text/html"
+        >>> request.get_preferred_type(["application/json", "text/plain"])
+        "application/json"
+        >>> request.get_preferred_type(["application/xml", "text/plain"])
+        None
 
-    Most browsers send ``Accept: */*`` by default, so this would return
-    ``True`` for all content types. Setting an explicit ``Accept`` header in
-    API requests can be useful for returning a different content type for those
-    consumers only. See :ref:`content-negotiation-example` of using
+    See :ref:`content-negotiation-example` for an example of using
```

Comment:
The diff here is a bit odd, I think we've moved this section out from under `HttpRequest.accepts(mime_type)`:
```See :ref:`content-negotiation-example` for an example of using ``accepts()`` ```
I'm not sure that's what was intended?

Perhaps you wanted a new link to ```:ref:`content-negotiation-example` ``` for the example of using `get_preferred_type()`?

Reply:
I've fixed the diff a little, so both `accepts` and `get_preferred_type` link to the example. My main intention was to make `get_preferred_type` the main API by moving most of the documentation onto it.